src/extension/tab/hook.ts

```diff
@@ -301,4 +301,19 @@ if (Array.isArray(preExisting)) {
   (preExisting as Array<ApolloClient<any>>).forEach(registerClient);
 }
 
-findClient();
```

Comment:
We no longer need this on initialization since we register the client immediately if is available before the script runs and handles when the value is assigned from a legacy client.